ActivityLauncherApp/src/main/java/de/szalkowski/activitylauncher/constant/Constants.java

```diff
@@ -0,0 +1,12 @@
+package de.szalkowski.activitylauncher.constant;
+
+public class Constants {
+    public static final String PREF_HIDE_PRIVATE = "hide_private";
```

Comment:
All these constants are related to preferences. Move to preferences package.

Reply:
They are related yes, but also used in different packages of the app, like the base activity or the IconLauncher class.